ClassTypes/PowerShell/HyperV/check-hyperv-vm/README.md

```diff
@@ -0,0 +1,37 @@
+# Check my Hyper-V VMs
+
+[Test-HypervVmConfig](./Test-HypervVmConfig.ps1) will verify you setup follows best practices for nested virtualization.  
+
+For all local users:
+
+- Verify user can use Hyper-V.
+
+For each Hyper-V client VM:
+
+- Verify the Hyper-V VM is **not** in a saved state.  
+- Verify the AutomaticStopAction is set to Stop.
+- Check the number of vCPUs.
+- Verify adequate memory assigned.
+- Verify variable memory enabled.
+- Verify disks are VHDX.
+
+For the host VM:
+
+- Verify DHCP role is not installed.
+- Verify sufficient free space on OS disk.
+
+## Usage
+
+In an Adminstrator PowerShell window, run:
```

Comment:
Might want to say that this should be run on the host VM (not from the client machine).  IE: start up the template, download the script to the template VM and run it)